EIPS/eip-1913.md

```diff
@@ -0,0 +1,277 @@
+---
+eip: 1913
+title: Casper PEPoW token for layer 2 solutions
+author: Wanseob Lim (@wanseob)
+discussions-to: https://github.com/ethereum/EIPs/issues/1913
+status: Draft
+type: Standards Track
+category: ERC
+created: 2019-04-05
+---
```

Comment:
If the interface below extends ERC-20, then I think here should be a `requires: 20` line.

Reply:
I've added the `requires` statement.